viewer/examples/headless.js

```diff
@@ -0,0 +1,119 @@
+/* global THREE */
+
+const { spawn } = require('child_process')
+const net = require('net')
+global.THREE = require('three')
+global.Worker = require('worker_threads').Worker
+const { createCanvas } = require('node-canvas-webgl/lib')
+
+const { Viewer, WorldView, getBufferFromStream } = require('..')
```

Comment:
why this example is not with the other examples ? Wouldn't it be better to have the examples in one place? Otherwise we should also move standalone and electron here to keep the logic.

Reply:
the idea was to have it all in the viewer/ folder as if it was its own lib, but yeah let's keep it all together for now, moved